### DISCUSSION TITLE
[Bug] log4j2 grpc report missing traceId

### DISCUSSION THREAD

**Discussion by andotorg** [apache/skywalking#9113] [Ideas]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

Java Agent (apache/skywalking-java)

### What happened

Added to the jvm parameter in the project.
-Dlog4j2.contextSelector=org.apache.logging.log4j.core.async.AsyncLoggerContextSelector
Traceid cannot be obtained as a result

### What you expected to happen

-Dlog4j2.contextSelector=org.apache.logging.log4j.core.async.AsyncLoggerContextSelector
after join and report traceid

### How to reproduce

jvm param add
-Dlog4j2.contextSelector=org.apache.logging.log4j.core.async.AsyncLoggerContextSelector

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
Are you going to submit a fix?

**Reply by andotorg** [reply]
https://github.com/apache/skywalking-java/pull/181

**Comment by andotorg** [comment]
yes, i try.

**Comment by wu-sheng** [comment]
I would like to move this into the discussion panel if you have not done. Issue panel for SkyWalking usually tracks things as release blocks or has owners fix them soon.

**Comment by andotorg** [comment]
Doing

**Comment by andotorg** [comment]
https://github.com/apache/skywalking-java/pull/181

**Comment by andotorg** [comment]
According to your suggestion, I resubmitted PR https://github.com/apache/skywalking-java/pull/182

**Comment by liguotian** [comment]


**Reply by andotorg** [reply]
thanks，for this problem, in Repair with version  V8.11.0
